src/domUpdates.js

```diff
@@ -22,17 +71,31 @@ const tagContainer = document.querySelector("#tagContainer");
 const saveRecipeButton = document.querySelector(".save-button");
 
 let currentUser;
+let users = [];
+let ingredients = [];
+let recipes = [];
 
 function initialize() {
-  generateRecipeCards();
-  populateAllRecipesPage();
-  // showFullRecipe();
-  // displayUserRecipes(userName)
-  currentUser = getRandomUser();
-  console.log("Current user: ", currentUser);
+  Promise.all([fetchUsers(), fetchIngredients(), fetchRecipes()])
+    .then(([fetchedUsers, fetchedIngredients, fetchedRecipes]) => {
+      users = fetchedUsers;
+      ingredients = fetchedIngredients;
```

Comment:
Note what is returned back when updating these global variables.  Is `fetchUsers` or `fetchIngredients` an array or an object?  If it's an object, potentially you could assign it the array value directly here.